### ISSUE TITLE
Again:  AxiosError: Request failed with status code 429, reconnect in  306s.

### ISSUE THREAD

**Issue by tobuser** [grzegorz914/homebridge-melcloud-control#85]
Now i have again login problems. I config 306 seconds but after hours i get this error:  Login error: AxiosError: Request failed with status code 429, reconnect in  306s.

**Comment by tobuser** [comment]
Please close this issues. I think the problem was initiate from a Home assistant integration

**Comment by bkarakashev** [comment]
I am getting the same error.
@tobuser what was the issue with Home Assistant integration? Could you please provide a bit more details?
Is there a conflict, I am using Home Assistant and Homebridge and I have the melcloud-control enabled in both places.

**Comment by tobuser** [comment]
The Melcloud error also occurs for me at 240s. I cannot yet say whether the problem is caused by Home Assistant. This is because I first had to completely delete various Melcloud integrations. I also have two Home Assistant installations at two locations, and I had to make sure that all Home Assistant integrations are deleted there as well. Then I had two installations of Melcloud Control at the two locations. All this can lead to the Melcloud error. Now I am testing the current Melcloud Control plugins. Unfortunately, some versions crash, which in turn leads to the Melcloud error. Let's see how it goes.
